user/waydroid/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "waydroid"
+pkgver = "1.4.3"
+pkgrel = 0
+build_style = "makefile"
+make_build_args = ["USE_SYSTEMD=0", "USE_NFTABLES=1"]
```

Comment:
this has to be make_install_args, currently it's just ignored

then you don't need the uninstall below for systemd either